packages/tx-ingestion/package.json

```diff
@@ -55,4 +55,4 @@
   "publishConfig": {
```

Comment:
@alexanderattar one bug that I noticed when messing around with this repo is that the compiler version is out of date & that was giving me errors when trying to test out my ERC20. I think the version should be `0.5.16-alpha.6`

Reply:
you mean  `@eth-optimism/solc` right?